baseplate/lib/metrics.py

```diff
@@ -303,12 +327,26 @@ def send(self, elapsed: float, sample_rate: float = 1.0) -> None:
         :param elapsed: The elapsed time in seconds to report.
 
         """
-        serialized = self.name + (f":{(elapsed * 1000.0):g}|ms".encode())
+        # serialized = self.name + (f":{(elapsed * 1000.0):g}|ms".encode())
+        serialized = (
+            self.name + (f":{(elapsed * 1000.0):g}|ms".encode()) + _format_tags(tags).encode()
```

Comment:
maybe write this like `self.name + f":{(elapsed * 1000.0):g}|ms{_format_tags(tags)}".encode()`? (also maybe worth benchmarking)